sensu-plugins-gpg.gemspec

```diff
@@ -28,15 +28,17 @@ Gem::Specification.new do |s|
   s.test_files             = s.files.grep(%r{^(test|spec|features)/})
   s.version                = SensuPluginsGpg::Version::VER_STRING
 
-  s.add_runtime_dependency 'sensu-plugin', '~> 1.2'
+  s.add_runtime_dependency 'sensu-plugin', '~> 2.0'
 
   s.add_development_dependency 'bundler',                   '~> 1.7'
   s.add_development_dependency 'codeclimate-test-reporter', '~> 0.4'
   s.add_development_dependency 'github-markup',             '~> 1.3'
+  s.add_development_dependency 'json',                      '~> 1.8'
```

Comment:
per https://github.com/adityapahuja/sensu-plugins-gpg/blob/ee6af8f9904258c8eb7d0bf97b70f68e73092a12/bin/check-all-gpg-keys-for-expiry.rb#L42 this would be a runtime dependency.

Reply:
I'd just remove the `json` dep since it gets pulled in from `sensu-plugin`.

Reply:
I have removed `json`.